src/game_interpreter.cpp

```diff
@@ -847,7 +851,10 @@ bool Game_Interpreter::OnFinishStackFrame() {
 	if (is_base_frame && event_id > 0) {
 		Game_Event* evnt = Game_Map::GetEvent(event_id);
 		if (!evnt) {
-			Output::Error("Call stack finished with invalid event id {}. This can be caused by a vehicle teleport?", event_id);
+			if (!Player::HasEasyRpgExtensions()) {
+				// Destroy Map Event triggers this sanity check
+				Output::Error("Call stack finished with invalid event id {}. This can be caused by a vehicle teleport?", event_id);
+			}
```

Comment:
Is there a way to maintain this check for all cases except for when the destroymapevent case triggers it?

Reply:
yes. I set now the base frame event_id on the foreground interpreter to 0 (when it is the deleted event).